services/iaasalpha/wait/wait.go

```diff
@@ -346,3 +349,53 @@ func DeleteVirtualIPWaitHandler(ctx context.Context, a APIClientInterface, proje
 	handler.SetTimeout(15 * time.Minute)
 	return handler
 }
+
+// ImageUploadWaitHandler will wait for server creation
+func ImageUploadWaitHandler(ctx context.Context, a APIClientInterface, projectId, imageId string) *wait.AsyncActionHandler[iaasalpha.Image] {
```

Comment:
Why not `UploadImageWaitHandler`?
More uniform with the other waiter names

Reply:
Also though about it, did it differently since there is no actual operation called `UploadImage`. But we can keep the uniformity yes